content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -31,7 +31,6 @@
 "Goy.Chat","An open source platform powered by Meteor uses the POWER infrastructure to test deployments on POWER8."
 "Hadoop_Spark","CI environment powered by a Jenkins server running Hadoop and Spark builds for including POWER into the Hyperledger fabric community to perform Continuous Integration for IBM Hyperledger fabric codebase."
 "Hortonworks Data Platform","A secure, enterprise-ready open source Apache Hadoop distribution based on a centralized architecture (YARN) now on POWER"
-"ICU","The International Components for Unicode project runs their build machines on a OSL hosted POWER VM"
```

Comment:
Can you please add this project to the former CSV file?

Reply:
fixed!